src/js/routes/result/Result.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import BackButton from '../../components/BackButton.jsx';
+
+const Result = (props) => (
```

Comment:
As far as I can see the table isn't going to support expanding rows ( unfortunately :( ). Golly, more reasons to try make my own table haha.

A result page might be the end result, comes back to getting a proper design done!